lib/ome/common/OMECommonConfig.cmake.in

```diff
@@ -1,12 +1,12 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(OMECompat REQUIRED)
-find_dependency(Boost 1.46 REQUIRED
+find_dependency(OMECompat)
+find_dependency(Boost 1.46
                 COMPONENTS date_time filesystem system iostreams
-                program_options regex)
-find_dependency(XercesC 3.0.0 REQUIRED)
```

Comment:
Is the drop of the `REQUIRED` qualifier required for the conditional import to work or is this related to the `find_dependency` limitations discussed in https://github.com/ome/ome-files-cpp/issues/85#issuecomment-332471884?

Reply:
It's unrelated to both. It's always been incorrect to use `REQUIRED` with `find_dependency`, but I didn't understand this at the time these files were written.  `find_dependency` adds `REQUIRED` for you if the top-level `find_package` call used `REQUIRED` otherwise it's not added and left optional.  It will work without the change, but the behaviour will be wrong.